01_fib/jimbotech/README.md

```diff
@@ -0,0 +1,15 @@
+# Fibonacci Number Generator
+## Overview
+As part of the ANZ "go" course the first lab is to produce a piece of software that will produce the Fibonacci series for fib(7). It was also an exercise in the tour.golang.org.
+
+Generating the series was implemented as "closure" which was a requirement in the tour-golang . There are other ways to solve this problem, but I kept it as an example of closure for later reference.
+
+The function maintains state and hence has to be called sequentially.
+
+
+
+
+> Written with [StackEdit](https://stackedit.io/).
```

Comment:
Great to see you putting together a ReadME file, I don't think anyone else has done it. Perhaps remove the part that is auto-generated by the tools. e.g. 
`> Written with [StackEdit](https://stackedit.io/).`

Reply:
StackEdit is an open source community and I feel I should acknowledge their efforts by crediting them in my work, so if it is ok, I think leaving in the credit is the right thing to do.

Reply:
Fair enough.